CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Error analysis package
+/erroranalysis @imatiach-msft
```

Comment:
looks like we need a backup here, otherwise Ilya may not be able to check in. LOL